src/averageAges.js

```diff
@@ -15,13 +17,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(man => (
```

Comment:
Maybe it will be better to rename this variable to something like filterByMen. The same situation for other functions.